src/components/package/show/components/PackageTitleList/PackageTitleList.test.js

```diff
@@ -0,0 +1,137 @@
+import { render } from '@folio/jest-config-stripes/testing-library/react';
+import userEvent from '@folio/jest-config-stripes/testing-library/user-event';
+
+import { createMemoryHistory } from 'history';
+import { PackageTitleList } from './PackageTitleList';
+import getAxe from '../../../../../../test/jest/helpers/get-axe';
+import Harness from '../../../../../../test/jest/helpers/harness';
+
+const records = [
+  {
+    id: '1',
+    attributes: {
+      isSelected: true,
+      name: 'Title name1',
+      managedCoverages: [{
+        'beginCoverage': '2005-01-01',
```

Comment:
Could you please remove ' (from 'beginCoverage': '2005-01-01', to beginCoverage: '2005-01-01',) for current line and below?